SillyBusinessInc/SillyBusinessGame
# Feat:#105 tail system rework

## Purpose of this PR:
Attacks work with animations now and now a tail has a has a ScriptableObject and that has a air and groundcombo, the attacks of a player are all prefabs now, so if a designer wants to change a combo they easily can. 
## How to test:
Go to the waffle ScriptableObject and change the combo (air or ground) with the prefab attacks, then it should still act correctly

### links: 
[Ticket](https://github.com/SillyBusinessInc/SillyBusinessGame/issues/105)
